<launch>
    <arg name="model_config" default="$(find pacejka_model)/config/model_params.yaml" />
    <arg name="simulator_config" default="$(find ros_simulator)/config/pacejka_car_simulator.yaml" doc = "Path to simulator.yaml config. "/>



    <!-- Load the Model -->
    <group ns="model">
        <rosparam command="load" file="$(arg model_config)" />
    </group>

    <node pkg="ros_simulator" name="ros_simulator" type="simulation_node" output="screen">
        <rosparam command="load" file="$(arg simulator_config)" />
    </node>


</launch>